src/main/java/fiji/plugin/trackmate/action/CloseGapsByLinearInterpolationAction.java

```diff
@@ -0,0 +1,157 @@
+    public static final String NAME = "Close gaps by introducing new spots";
+
+    public static final String KEY = "CLOSE_GAPS_BY_LINEAR_INPERPOLATION";
+    public static final String INFO_TEXT = "<html>" +
+            "This action closes gaps in tracks by introducing new spots. The spots positions and size are calculated using linear interpolation." +
+            "</html>";
+
+    @Override
+    public void execute(final TrackMate trackmate) {
+        final Model model = trackmate.getModel();
+
+
+        Set<Integer> trackIDs = model.getTrackModel().trackIDs(true);
+
+        // go through all tracks
+        for (Integer trackID : trackIDs)
+        {
+            Set<Spot> spots = model.getTrackModel().trackSpots(trackID);
+            if (spots != null) {
+                model.beginUpdate();
+
+                // go through all spots, search for gaps
+                for (Spot currentSpot : spots) {
+                    int currentFrame = currentSpot.getFeatures().get("FRAME").intValue();
+
+                    Spot nextSpot = null;
+                    int nextFrame = 0;
+
+                    // find the next following spot
+                    for (Spot futureSpot : spots) {
+                        int futureFrame = futureSpot.getFeatures().get("FRAME").intValue();
+                        if (futureFrame > currentFrame) {
+                            if (nextSpot == null || nextFrame > futureFrame) {
+                                nextSpot = futureSpot;
+                                nextFrame = futureFrame;
+                            }
+                        }
+                    }
+
+                    // if a following spot was found and there is a gap between current and next
+                    if (nextSpot != null && (nextFrame - currentFrame > 1)) {
+
+                        double[] currentPosition = new double[3];
+                        double[] nextPosition = new double[3];
+
+                        nextSpot.localize(nextPosition);
+                        currentSpot.localize(currentPosition);
+
+                        model.removeEdge(currentSpot, nextSpot);
+
+                        Spot formerSpot = currentSpot;
+                        for (int f = currentFrame + 1; f < nextFrame; f++) {
+                            double weight = (double) (nextFrame - f) / (nextFrame - currentFrame);
+
+
+                            double[] position = new double[3];
+                            for (int d = 0; d < currentSpot.numDimensions(); d++) {
+                                position[d] = weight * currentPosition[d] + (1.0 - weight) * nextPosition[d];
+                            }
+
+                            RealPoint rp = new RealPoint(position);
+
+                            Spot newSpot = new Spot(rp, 0, 0);
+
+                            // Set some properties of the new spot
+                            interpolateFeature(newSpot, currentSpot, nextSpot, weight, "RADIUS");
```

Comment:
Please use the constants defined in the analyzer of the Spot class instead of String literals. 
For example, use `Spot.QUALITY`, `Spot.RADIUS`, etc...